pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -189,12 +190,13 @@ def write_pep0(self, peps: list[PEP], header: str = HEADER, intro: str = INTRO,
             if status in HIDE_STATUS:
                 continue
             if status == STATUS_ACCEPTED:
-                msg = "    A - Accepted (Standards Track only) or Active proposal"
+                msg = "* A - Accepted (Standards Track only) or Active proposal"
             else:
-                msg = f"    {status[0]} - {status} proposal"
+                msg = f"* {status[0]} - {status} proposal"
```

Comment:
As mentioned above, all these lines can be removed, as the above line replaces this special casing.